src/arrayMethodJoin.js

```diff
@@ -5,8 +5,27 @@
  *
  */
 function applyCustomJoin() {
-  [].__proto__.join2 = function() {
-    // write code here
+  [].__proto__.join2 = function(symbol) {
+    let length = this.length;
+    // if recieved empty array , returns empty string
+    let result = length
+      ? `${this[0]}`
+      : '';
+
+    // typeof argument is undefined returns join by ','
```

Comment:
Same here, everything is clear, no need for a comment.

Reply:
Done